packages/utxo-lib/README.md

```diff
@@ -35,6 +36,8 @@ Differences are explicitly described in each file.
 -   added `coinselect` and `compose` modules (origin source: https://github.com/trezor/hd-wallet/tree/master/src/build-tx)
 -   removed unused modules: `block`, `classify`, `ecpair`, `psbt`.
 
-### Publishing
+## Publishing
+
+This package is published to npm registry because it is a dependency of [@trezor/connect](https://github.com/trezor/trezor-suite/issues/5440) which can be installed as a standalone package.
```

Comment:
```suggestion
This package is published to the npm registry because it is a dependency of [@trezor/connect](https://github.com/trezor/trezor-suite/issues/5440) which can be installed as a standalone package.
```